### DISCUSSION TITLE
Cohesive elements multiscale tessellation

### DISCUSSION THREAD

**Discussion by ravikiranbollineni** [neperfepx/neper#190] [Q&A] [answered]
Hello Romain,

I am trying to create cohesive elements in multiscale tessellation using the code below, 

```
neper -T -n 30::from_morpho -id 1::1 -reg 1 -dim 3 -domain 'cube(.1,.1,.001)' -morpho "voronoi::lamellar(w=0.001)" -morphooptistop "nlopt_reps=10000" -format geo,tess -o cohesive112619
neper -M cohesive112619.tess -dim 3 -pl 100  -interface cohesive -format inp,msh
```

Using this code I am able to create cohesive elements between the lamellar and also at the grain boundaries. But I only want cohesive elements to be present at the grain boundaries not in between the lamellar.

can you help in this regards?

Thank you,
Ravi.

**Comment by rquey** [comment, marked as answer]
Use `-interface cohesive::continuous`.
